fastlane/Fastfile

```diff
@@ -512,34 +576,36 @@ UPLOAD_TO_PLAY_STORE_JSON_KEY = File.join(PROJECT_ROOT_FOLDER, '.configure-files
   end
 
   def screenshots_directory
-    File.join(Dir.pwd, "screenshots")
+    File.join(Dir.pwd, 'screenshots')
   end
 
   #####################################################################################
   # Private lanes
   #####################################################################################
-  main_strings_path = "./Simplenote/src/main/res/values/strings.xml"
-  update_strings_path = "./fastlane/resources/values/"
+  main_strings_path = './Simplenote/src/main/res/values/strings.xml'
+  update_strings_path = './fastlane/resources/values/'
 
-  private_lane :delete_old_changelogs do | options |
-    Dir.glob("./metadata/android/*/").each do | folder |
-      Dir["#{folder}changelogs/*"].each do | file |
-        File.delete(file) if Integer(File.basename(file, ".*")) < Integer(options[:build]) rescue puts "Cannot delete file #{file}"
+  private_lane :delete_old_changelogs do |options|
+    Dir.glob('./metadata/android/*/').each do |folder|
+      Dir["#{folder}changelogs/*"].each do |file|
+        File.delete(file) if Integer(File.basename(file, '.*')) < Integer(options[:build])
+      rescue StandardError
+        puts "Cannot delete file #{file}"
       end
     end
   end
 
-  private_lane :add_us_release_notes do | options |
-    en_release_notes_path  = Dir.pwd + "/.." + "/Simplenote/metadata/release_notes.txt"
-    File.open(options[:release_notes_path], "a") { |f|
-      f.puts("<en-US>")
+  private_lane :add_us_release_notes do |options|
+    en_release_notes_path = "#{Dir.pwd}/../Simplenote/metadata/release_notes.txt"
```

Comment:
`File.join`

Reply:
Fixed in https://github.com/Automattic/simplenote-android/commit/3267c3d07bb4efb667be614d849bb73e9b2b9ac3